examples/pytorch/README.md

```diff
@@ -0,0 +1,21 @@
+# Pytorch example script
+
+To run this script you will need to clone the jean-zay repo in your `$WORK`
+dir, then:
+```
+cd $WORK &&\
+git clone https://github.com/jean-zay-users/jean-zay-doc.git
+```
+
+After that, you can just launch the batch job (single GPU) via:
```

Comment:
Minor detail, is something I learned from Software Carpentry is to avoid "just". It is not useful and can be detrimental (i.e. demotivating) if the person does not understand what you mean.

Reply:
Actually re-reading this, maybe you want to mention that it launches a job array (i.e. the equivalent of 10 jobs with different learning rate parameters)

Reply:
I edited this text.